exporter/elasticsearchexporter/config.go

```diff
@@ -217,10 +223,14 @@ func (cfg *Config) Validate() error {
 		}
 	}
 
-	if cfg.Index == "" {
+	if cfg.Index == "" && cfg.LogsIndex == "" || cfg.TracesIndex == "" {
```

Comment:
Weird that we have default index value `logs-generic-default` that is never being used because the field is required. Do you mind submitting an issue to address that separately. We need to either remove the default value or make the config option not required

Reply:
The logic here was not clear before to me, but if we had the default value by `createDefaultConfig` method, should be no need to detect it here.

Reply:
I think we can remove this logical.